metadata.json

```diff
@@ -17,8 +17,6 @@
     {
       "operatingsystem": "Windows",
       "operatingsystemrelease": [
-        "2012",
-        "2012 R2",
```

Comment:
I don’t think we can drop support whilst agent still supports these platforms.
If it is the case I think we should at least have this in a separate PR so it has a dedicated change log entry/release note.

Reply:
I have removed it as we don't have image available, here is from nightly build :
```
Warning: Cannot find image for Windows-2012
Warning: Cannot find image for Windows-2012 R2
```